### ISSUE TITLE
Mistake with rate limit for status updates in Emscripten shell file?

### ISSUE THREAD

**Issue by nandenjin** [openframeworks/openFrameworks#7152]
The template file for Emscripten build contains the logic which makes no sense:

https://github.com/openframeworks/openFrameworks/blob/d04f836a2e04f6de7bd3d04365ea8b7b5dfabcc7/libs/openFrameworksCompiled/project/emscripten/template.html#L183-L184

It compares `Date.now()` and `Date.now()`, with no process between them.

The template provided by emscripten team has [very similar but different code](https://github.com/emscripten-core/emscripten/blob/000cedf1d23860787f0839e3b320a8f2aa456039/src/shell_minimal.html#L110-L111). I think this is correct one:

```html
var now = Date.now();
if (m && now - Module.setStatus.last.time < 30) return; // if this is a progress update, skip it if too soon
```

**Comment by nandenjin** [comment]
I can submit PR for it if my question is correct and I asked to do so 👋

**Comment by ofTheo** [comment]
Oh right yeah - we would need to compare against a last updated time instead of the time now twice. 
If you want to do a PR ( please check that it works first ) that would be amazing! 🙂

**Comment by nandenjin** [comment]
It seems to be fixed in #7056 . Thanks @Jonathhhan
